ios/runblockingTests/runblockingTests.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  runblockingTests.swift
+//  runblockingTests
+//
+//  Created by Janek Góral on 31/01/2020.
+//  Copyright © 2020 gogoapps. All rights reserved.
```

Comment:
Let's remove the default comments from Xcode.

Reply:
Ok, I'll do it.